src/Mod/PartDesign/App/FeatureLoft.cpp

```diff
@@ -110,13 +110,24 @@ App::DocumentObjectExecReturn *Loft::execute(void)
         for(TopoDS_Wire& wire : wires)
             wiresections.emplace_back(1, wire);
 
-        for(App::DocumentObject* obj : multisections) {
+        for(auto obj : multisections) {

```

Comment:
Just a style note: after if, for etc. we have a space
(This occurs at several positions in the PR.)

Reply:
Thanks for the note, but I think looking at the rest of the document (for example the for loop just above in line 110), I'd like to keep this as is to maintain style consistency.

Reply:
>  I'd like to keep this as is to maintain style consistency.

We try to fix the style whenever we edit code. So when you see a "for(" you can just change it to "for (".

Reply:
OK, leave it as it is. I will do this when it is merged since MSVC does this automatically for me with just few clicks.

Reply:
Looking at it again seems the entire file uses a mix of space and no space style, so just added the spaces.